client/app/pods/dashboard/template.hbs

```diff
@@ -2,15 +2,25 @@
 {{flash-messages classNames="flash-messages--below-control-bar"}}
 
 <div id="tahi-container">
-  <section id="dashboard-my-submissions">
+
+  <section class="dashboard-section">
+    {{#if pendingInvitations.length}}
+      <h2 class="welcome-message">
+        You have {{model.invitations.length}} {{make-plural pendingInvitations.length "invitation" "invitations"}}.
```

Comment:
The number and the pluralized word are looking at 2 different things (`model.invitations` vs `pendingInvitations`). It should be consistentificationalized. -- AC + CT